config_reader.py

```diff
@@ -0,0 +1,15 @@
+import imp
```

Comment:
Add a short docstring. Maybe just something like:
"mechanism for dynamically loading config file/module based on command line argument"